Reason: Try sndio before alsa (and pulse). Alsa is always available
on Void systems, so openal will try to use it, even if sndio is setup,
furthermore the alsa sndio plugin does not work well with OpenAL
---
--- alc/alc.cpp	2020-01-24 08:28:45.000000000 +0100
+++ -	2020-07-25 14:13:19.805487926 +0200
@@ -163,6 +163,9 @@
 #ifdef HAVE_JACK
     { "jack", JackBackendFactory::getFactory },
 #endif
+#ifdef HAVE_SNDIO
+    { "sndio", SndIOBackendFactory::getFactory },
+#endif
 #ifdef HAVE_PULSEAUDIO
     { "pulse", PulseBackendFactory::getFactory },
 #endif
@@ -181,9 +184,6 @@
 #ifdef HAVE_SOLARIS
     { "solaris", SolarisBackendFactory::getFactory },
 #endif
-#ifdef HAVE_SNDIO
-    { "sndio", SndIOBackendFactory::getFactory },
-#endif
 #ifdef HAVE_OSS
     { "oss", OSSBackendFactory::getFactory },
 #endif
